CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.2.1
+
+### Bug fixes
+
+* Set native library version to ReLinker (#3775)
```

Comment:
Perhaps rephrase it so it is more meaningful to users?

`Added version number to the native library, preventing ReLinker from accidentally loading old code` ?